spec/services/measurements_creator_spec.rb

```diff
@@ -0,0 +1,60 @@
+require "spec_helper"
+
+describe MeasurementsCreator do
+  let(:measurements_attributes) { [{:longitude=>25.4356212, :latitude=>56.4523456, :time=>"2016-05-11T17:09:02", :milliseconds=>925, :measured_value=>59.15683475380729, :value=>59.15683475380729}] }
+
+  it "creates a mesurement" do
+    stream = create_stream!
+    MeasurementsCreator.call(stream, measurements_attributes)
+
+    expect(Measurement.count).to eq(1)
+  end
+
+  it "creates a mesurement with utc time for fixed sessions" do
```

Comment:
Personally I follow the convention of putting context first in the test name. In this case it would be "when session is fixed it creates a measurement with utc time". But I'm fine with this. Also, watch out for the spelling in measurement.